src/equations/serre_green_naghdi_1d.jl

```diff
@@ -381,7 +380,7 @@ end
 # Discretization that conserves
 # - the total water mass (integral of h) as a linear invariant
 # - the total momentum (integral of h v) as a nonlinear invariant
-# - the total energy
+# - the total modified energy
```

Comment:
Can you add one line above that the total momentum is only conserved for constant bathymetry?